src/Classes/ServiceAPI/MyRadio_Album.php

```diff
@@ -81,13 +86,13 @@ protected function __construct($recordid)
         $this->media = $result['media_descr'];
         $this->format = $result['format_descr'];
         $this->record_label = $result['recordlabel'];
-        $this->date_added = strtotime($result['dateadded']);
-        $this->date_released = strtotime($result['datereleased']);
+        $this->date_added = (int) $result['dateadded'];
```

Comment:
http://php.net/manual/en/function.pg-field-type.php

Oh.

I have genuinely never thought to ask that question.